src/actions/setPost.tsx

```diff
@@ -0,0 +1,16 @@
+import { db } from '../firebase';
+
+interface FeedPostProps {
+  ownerUid: string | undefined | null;
+  displayName: string | undefined | null;
```

Comment:
it could be either null or undefined?? it's weird - it refers to the three properties